config/Sage/jobs.txt

```diff
@@ -12,12 +12,12 @@ Chief Medical Officer=1,1
 Assistant=-1,-1
 
 Quartermaster=1,1
-Cargo Technician=4,2
-Shaft Miner=3,3
+Cargo Technician=6,2
+Shaft Miner=4,3
```

Comment:
Make sure Mining has enough equipment for everyone if you increasing the starting value.

Reply:
the mining station has an additional 3 lockers, you could have a total of 6